setup.cfg

```diff
@@ -7,6 +7,6 @@ with-doctest=1
 doctest-extension=rst
 
 [build_sphinx]
-source-dir = docs/
-build-dir  = docs/_build
+source-dir = doc/
```

Comment:
> But now that we can call sphinx-build from any place I thought to make use of this. 

I think that is what I have an issue with, `python setup.py build_sphinx` should build the doc in the same place as `cd doc && make html` (principle of least surprise). In circle.yml use explicit sphinx-build commands to test out different locations and theme.